deployments/ansible/molecule/default/Dockerfile.j2

```diff
@@ -1,9 +1,5 @@
 {% if item.image not in ["opensuse12", "opensuse15"] %}
 FROM geerlingguy/docker-{{ item.image }}-ansible:latest
-{% if item.image == "centos8" %}
-RUN sed -i 's|\$releasever|8-stream|g' /etc/yum.repos.d/CentOS*.repo
-RUN dnf install -y --allowerasing centos-stream-release
-{% endif %}
 {% else %}
```

Comment:
Was this change on purpose?

Reply:
It was a workaround until the [centos8 image](https://hub.docker.com/r/geerlingguy/docker-centos8-ansible) was updated for the new `8-stream` repo.  Otherwise the docker build would fail.